contracts/contracts/vault/VaultStorage.sol

```diff
@@ -220,6 +221,8 @@ contract VaultStorage is Initializable, Governable {
     /// @notice Mapping of withdrawal request indices to the user withdrawal request data
     mapping(uint256 => WithdrawalRequest) public withdrawalRequests;
 
+    uint256 constant CLAIM_DELAY = 30 minutes;
```

Comment:
VaultStorage is also used by OUSD which doesn't have the withdrawal queue. I'd move this constant to `OETHVaultCore` which is where the request and claim functions are.